test/testcommon/src/main/java/io/pravega/test/common/TestUtils.java

```diff
@@ -30,6 +29,13 @@
     private static final AtomicInteger NEXT_PORT = new AtomicInteger(new Random().nextInt(MAX_PORT_COUNT));
 
     /**
+     * We use this as a source of alphanumeric characters when generating random strings. Characters will be selected at
+     * random from this string.
+     */
+    private static final char[] ALPHANUMERIC_CHARS = "0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz".toCharArray();
+    private static final Random RANDOM = new Random(0);
```

Comment:
I was thinking of that. But this is for a test. In tests, I would like things to be sort of reproducible. If I have a constant seed, then this thing will always generate the same values in the same order. However, there is no guarantee as to the order in which things will be executed, so essentially it is not consistent.

I think either way it's fine. It doesn't matter if it's 0 or `System.currentTimeMillis()` simply because it's a static field shared across all the tests in the process, and there's no guarantee as to the execution order of those tests.

Reply:
This should be annotated with @GuardedBy("RANDOM")

Reply:
duh! Fixed.